docker-container/docker_env

```diff
@@ -0,0 +1,16 @@
+MYSQL_HOST=db
+MYSQL_ROOT_PASSWORD=myrootpassword
+MYSQL_USER=magento
+MYSQL_PASSWORD=magento
+MYSQL_DATABASE=magento
+
+MAGENTO_LOCALE=en_GB
+MAGENTO_TIMEZONE=Pacific/Auckland
```

Comment:
I think this being the default timezone will be very unintuitive for people just picking up this container without fully examining it.